seed/static/seed/js/controllers/inventory_detail_controller.js

```diff
@@ -132,9 +132,9 @@ angular.module('BE.seed.controller.inventory_detail', [])
         })
       }
       $scope.order_historical_items_with_scenarios($scope.historical_items)
-      $scope.format_epoch = (epoch) => {
-        return moment(epoch).format('YYYY/MM/DD')
-      }
+      $scope.format_epoch = (epoch) => moment(epoch).format('YYYY/MM/DD');
+      $scope.replace_all =  (date, a, b) => date.replace(new RegExp(a, 'g'), b);
```

Comment:
ideally, we could use `string.replaceAll('-', '/')` but since `replaceAll` was introduced in 2021 I was guarding against any old browser javascript versions